src/Angor/Avalonia/AngorApp/UI/Sections/Founder/ProjectDetails/FounderProjectDetailsViewModel.cs

```diff
@@ -34,14 +36,26 @@ public FounderProjectDetailsViewModel(ProjectId projectId, IProjectAppService pr
 
     private static object CreateContent(ProjectId projectId, ProjectStatus status, IFounderAppService founderAppService, UIServices uiServices, IWalletContext walletContext)
     {
+        // todo: fix this so that we can load approve
+        // and claim at the same time for fund type projects
+        // this is an ugly hack to bypass the static object in the method signature
```

Comment:
We should not have "ugly hack" in our code, perhaps add a method before this one for the required actions and call it in a select before the call to CreateContent